src/scripts.js

```diff
@@ -95,24 +116,32 @@ function renderBookings() {
   bookingsSection.innerHTML += `<p>Total Spent: $${hotel.getTotal().toFixed(2)}` 
 }
 
-function showAvailable(date) {
+function showAvailable() {
+  const date = dateInput.value.split('-').join('/');
+  console.log(date)
+  const type = typeSelect.value;
+  console.log(hotel.bookings)
+  hotel.findAvailable(date);
+  filterAvailable(type);
```

Comment:
Maybe it's working fine and I'm missing something but I think you'll need to include this: availableSection.innerHTML = ''
at the beginning of this function otherwise if the user was to search for more than one date the available reservations will stack up on top of each other and you'll just keep adding to what was put in that section before.